ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb

```diff
@@ -31,7 +31,7 @@
     "repo": "snippets_housekeeping.ipynb"
```

Comment:
<p>Learn [which accelerators are available in your region](https://cloud.google.com/vertex-ai/docs/general/locations#accelerators).</p><p> </p><p>Do we still need the TF 2.3 note? It's OK to keep it, it just depends on how commonly you think it's still needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>fixed.</p><p>I will continue with the note for the time being.</p>